api/environments/dynamodb/types.py

```diff
@@ -77,8 +77,13 @@ class IdentityOverrideV2(BaseModel):
     environment_id: str
     document_key: str
     environment_api_key: str
+    identifier: str
     feature_state: FeatureStateModel
 
+    @property
+    def identity_uuid(self) -> str:
+        return self.document_key.rsplit(":", maxsplit=1)[1]
+
```

Comment:
1. I'd rather see this code in a mapper somewhere.
2. The code itself seems a bit hacky. Can't we just store the identity_uuid?

Reply:
We could definitely just store the `identity_uuid`, yep. I think that's the better option here.

Reply:
This is done ^ ✅

Reply:
Please modify `map_engine_feature_state_to_identity_override` and its calls.

Reply:
It's already done [here](https://github.com/Flagsmith/flagsmith/pull/3116/files#diff-1a5f21351538565333f52d2a54d530cedc6e3ac8bf95e9b915a83622bc5264b3R103) 🤔